server.js

```diff
@@ -19,26 +32,54 @@ app.get("/contacts", function(req, res)
        console.log("Connected to db.mean-tut-dev.contacts."); 
     });
     
-    var contactSchema = mongoose.Schema
+    var contacts = Contact.find({}).lean().exec(function(err, contact)
+    {
+        if(err)
+        {
+            console.log(err);    
+        }
+        
+        res.json(contact);
+    });
+    
+    mongoose.disconnect();
```

Comment:
This is required (I believe) for expected behavior. This will probably end up being abstracted out anyways, so I'm not too worried with the specific best-practices right now.